### PR TITLE
Updated example codes

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

Issue link:#1490

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/references/pysdk_api_reference.md [hunk 1]
```diff
@@ -137,8 +137,12 @@ If `ConflictType` is not set, it defaults to `Error`.
 
 ### Returns
 
-- Success: A table object. 
-- Failure: `Exception`
+- Success: A `LocalDatabase` object in Python module mode or a `RemoteDatabase` object in client-server mode. Both objects contain these attributes:
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:140]: Return A RemoteDatabase object.

File: docs/references/pysdk_api_reference.md [hunk 2]
```diff
@@ -271,8 +275,11 @@ Name of the database. Must not be empty.
 
 ### Returns
 
-- Success: A database object.  
-- Failure: `Exception`
+- Success: A `LocalDatabase` object in Python module mode or a `RemoteDatabase` object in client-server mode. Both objects contain these attributes:
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:278]: Same as above

File: docs/references/pysdk_api_reference.md [hunk 3]
```diff
@@ -363,8 +370,12 @@ If `ConflictType` is not set, it defaults to `Error`.
 
 ### Returns
 
-- Success: A table object. 
-- Failure: `Exception`
+- Success: A `LocalTable` object in Python module mode or a `RemoteTable` object in client-server mode. Both objects contain these attributes:
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:373]: Similar to CreateDatabase / GetDatabase return type.

File: docs/references/pysdk_api_reference.md [hunk 4]
```diff
@@ -624,29 +639,35 @@ An `IndexInfo` structure contains three fields,`column_name`, `index_type`, and
       - `"64"`
       - `"128"`
     - `"pq_subspace_bits"`: *Required*
-      - `"8"` (Recommended)
+      - `"8"` (recommended)
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:642]: pq_subspace_num should only recommend one value.

File: docs/references/pysdk_api_reference.md [hunk 5]
```diff
@@ -624,29 +639,35 @@ An `IndexInfo` structure contains three fields,`column_name`, `index_type`, and
...
       - `"16"`
   - Parameter settings for a full text index: 
-    - `"ANALYZER"`: *Optional* - Defaults to `"standard"`
+    - `"ANALYZER"`: *Optional* 
+      - `"standard"`: (Default) Standard
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:646]: What is standard?

File: docs/references/pysdk_api_reference.md [hunk 6]
```diff
@@ -624,29 +639,35 @@ An `IndexInfo` structure contains three fields,`column_name`, `index_type`, and
...
-    No parameters required for now. 
+  - Parameter settings for a secondary index:  
+    No parameters are required. For now, use an empty list `[]`.
   - Parameter settings for a BMP index: 
-    - `block_size`: *Optional* - The size of the block in a BMP index. Range: `"1"` ~ `"256"`. Defaults to 16.
+    - `block_size`: *Optional* - The size of the block in a BMP index. Range: `"1"` ~ `"256"`. Defaults to `"16"`.
     - `"compress_type"`: *Optional*  
       - `"compress"`: (Default) Store the block max in sparse format. Works best with small block size situations.
-      - `"raw"`: 
+      - `"raw"`: Store the block max without compression.
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:663]: What is block max?

File: docs/references/pysdk_api_reference.md [hunk 7]
```diff
@@ -702,33 +731,55 @@ table_obj.create_index(
...
+        infinity.index.IndexInfo(
             "c1",
-            IndexType.Hnsw,
+            infinity.index.IndexType.Hnsw,
             [
-                index.InitParameter("M", "16"),
-                index.InitParameter("ef_construction", "50"),
-                index.InitParameter("ef", "50"),
-                index.InitParameter("metric", "l2")
+                infinity.index.InitParameter("M", "16"),
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:747]: infinity.index.InitParameter("M", "16"), => InitParameter("M", "16"),

File: docs/references/pysdk_api_reference.md [hunk 8]
```diff
@@ -702,33 +731,55 @@ table_obj.create_index(
...
+    ],
+    None
+)
+```
+
 ```python
 from infinity.index import IndexType
-# Create a table named "test_index_fulltext" with a vector column "c1"
-table_obj = db_obj.create_table("test_index_fulltext", {"doctitle": {"type": "varchar"}, "docdate": {"type": "varchar"}, "body": {"type": "varchar"}}, None)
+from infinity import index
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:780]: One index includes three columns

Comment (JinHai-CN) [reply docs/references/pysdk_api_reference.md:780]: Multiple column index only works for full-text index.

File: docs/references/pysdk_api_reference.md [hunk 9]
```diff
@@ -620,33 +629,41 @@ An `IndexInfo` structure contains three fields,`column_name`, `index_type`, and
...
     - `"pq_subspace_bits"`: *Required*
-      - `"8"` (Recommended)
+      - `"8"` (recommended)
       - `"16"`
-  - Parameter settings for a full text index: 
-    - `"ANALYZER"`: *Optional* - Defaults to `"standard"`
+  - Parameter settings for a full-text index: 
+    - `"ANALYZER"`: *Optional* 
+      - `"standard"`: (Default) Standard analyzer, segmented by tokens, lowercase processing, provides stemming outputs.
+      - `"chinese"`: Chinese
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:641]: Simplified Chinese

File: docs/references/pysdk_api_reference.md [hunk 10]
```diff
@@ -672,51 +689,178 @@ If `ConflictType` is not set, it defaults to `Error`.
 
 ### Returns
 
-A structure containing the following attributes:
+A `LocalQueryResult` object in Python module mode or a `CommonResponse` object in client-server mode. Both objects contain these attributes:
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:692]: A structure containing the following attributes:

File: docs/references/pysdk_api_reference.md [hunk 11]
```diff
@@ -672,51 +689,178 @@ If `ConflictType` is not set, it defaults to `Error`.
...
+from infinity import index
+# Create a table named "test_index_fulltext" with a varchar column "body"
+table_obj = db_obj.create_table("test_index_fulltext", {"body": {"type": "varchar"}}, None)
+# Create a full-text index named "my_index" on column "body" with default parameter settings:
+# - "ANALYZER": "standard"
+table_obj.create_index(
+    "my_index",
+    [
+        IndexInfo(
+          "body",
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:764]: intent isn't aligned with other line.

File: docs/references/pysdk_api_reference.md [hunk 12]
```diff
@@ -672,51 +689,178 @@ If `ConflictType` is not set, it defaults to `Error`.
...
+from infinity import index
+# Create a table named "test_index_fulltext" with a varchar column "body"
+table_obj = db_obj.create_table("test_index_fulltext", {"body": {"type": "varchar"}}, None)
+# Create a full-text index named "my_index" on column "body"
+# Setting "ANALYZER" to "standard" (same as the above)
+table_obj.create_index(
+    "my_index",
+    [
+        IndexInfo(
+          "body",
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:786]: intent issue

File: docs/references/pysdk_api_reference.md [hunk 13]
```diff
@@ -725,36 +869,72 @@ table_obj.create_index(
...
+table_obj = db_obj.create_table("test_index_secondary", {"c1": {"type": "varchar"}}, None)
+# Create a secondary index named "my_index" on column "c1"
 table_obj.create_index(
     "my_index",
     [
-        index.IndexInfo("body", IndexType.FullText, []),
-        index.IndexInfo("doctitle", IndexType.FullText, []),
-        index.IndexInfo("docdate", IndexType.FullText, []),
+        IndexInfo(
+          "c1",
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:886]: Intent issue.

**Comment by writinwaters** [comment]
Issues put to rest.

**Pull request merged**